mifosng-provider/src/integrationTest/java/org/mifosplatform/integrationtests/OfficeIntegrationTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.mifosplatform.integrationtests;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mifosplatform.integrationtests.common.OfficeDomain;
+import org.mifosplatform.integrationtests.common.OfficeHelper;
+import org.mifosplatform.integrationtests.common.Utils;
+
+import com.jayway.restassured.builder.RequestSpecBuilder;
+import com.jayway.restassured.builder.ResponseSpecBuilder;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.specification.RequestSpecification;
+import com.jayway.restassured.specification.ResponseSpecification;
+
+public class OfficeIntegrationTest {
+
+	private ResponseSpecification responseSpec;
+	private RequestSpecification requestSpec;
+
+	@Before
+	public void setup() {
+		Utils.initializeRESTAssured();
+		this.requestSpec = new RequestSpecBuilder().setContentType(
+				ContentType.JSON).build();
+		this.requestSpec
+				.header("Authorization",
+						"Basic "
+								+ Utils.loginIntoServerAndGetBase64EncodedAuthenticationKey());
+		this.responseSpec = new ResponseSpecBuilder().expectStatusCode(200)
+				.build();
+	}
+
+	@Test
+	public void testOfficeModification() {
```

Comment:
This test case is failing on the Integration test server. Details at https://openmf.ci.cloudbees.com/job/MIFOSX%20INTEGRATION%20TEST/981/testReport/junit/org.mifosplatform.integrationtests/OfficeIntegrationTest/testOfficeModification/